gems/sorbet-runtime/lib/types/private/methods/modes.rb

```diff
@@ -2,12 +2,24 @@
 # typed: true
 
 module T::Private::Methods::Modes
-  def self.standard; 'standard'; end
-  def self.abstract; 'abstract'; end
-  def self.overridable; 'overridable'; end
-  def self.override; 'override'; end
-  def self.overridable_override; 'overridable_override'; end
-  def self.untyped; 'untyped'; end
+  def self.standard
+    'standard'
+  end
+  def self.abstract
+    'abstract'
+  end
+  def self.overridable
+    'overridable'
+  end
+  def self.override
+    'override'
+  end
+  def self.overridable_override
+    'overridable_override'
+  end
+  def self.untyped
+    'untyped'
+  end
```

Comment:
😢

Reply:
@jez what single-line method defs do you want to keep? Where the body is just a literal? Since I was kind of just looking for something to do anyway, maybe I'll submit a PR to open-source Rubocop to add that as an option...

Reply:
yeah not sure, was kind of just hoping for "whatever Stripe's codebase allows," which after a quick grep looks like it's doing this:

```yaml
# -- .rubocop.yml --
# Super simple single line methods are totally fine
Style/SingleLineMethods:
  Enabled: false
```